homeassistant/components/camera/__init__.py

```diff
@@ -128,8 +129,8 @@ def async_snapshot(hass, filename, entity_id=None):
     raise HomeAssistantError('Unable to get image')
 
 
-@asyncio.coroutine
-def async_setup(hass, config):
+@bind_hass
```

Comment:
Setup should not have bind Hass. Bind Hass is only needed for helper methods that other components might call, like get image